### PR TITLE
Shift ffmpeg logs from debug to Information

### PR DESCRIPTION
**Pull request by Artiume**
Ffmpeg logs shouldn't be a debug level event.

**Changes**

**Issues**

### CODE REVIEW

**Comment by dkanada** [comment]
That's not how log levels work, error messages can only be used for actual errors. This might work with the info level though.

File: MediaBrowser.Api/Playback/BaseStreamingService.cs [hunk 1]
```diff
@@ -281,7 +281,7 @@ private async Task AcquireResources(StreamState state, CancellationTokenSource c
                 throw;
             }
 
-            Logger.LogDebug("Launched ffmpeg process");
+            Logger.LogError("Launched ffmpeg process");
```
Comment (Artiume) [inline MediaBrowser.Api/Playback/BaseStreamingService.cs]:
```suggestion
            Logger.LogInformation("Launched ffmpeg process");
```

File: MediaBrowser.Api/Playback/BaseStreamingService.cs [hunk 2]
```diff
@@ -311,7 +311,7 @@ private async Task AcquireResources(StreamState state, CancellationTokenSource c
             {
                 StartThrottler(state, transcodingJob);
             }
-            Logger.LogDebug("StartFfMpeg() finished successfully");
+            Logger.LogError("StartFfMpeg() finished successfully");
```
Comment (Artiume) [inline MediaBrowser.Api/Playback/BaseStreamingService.cs]:
```suggestion
            Logger.LogInformation("StartFfMpeg() finished successfully");
```

**Comment by Artiume** [comment]
fixed,

File: MediaBrowser.Api/Playback/BaseStreamingService.cs (hunk 2 above)
Comment (Bond-009) [reply MediaBrowser.Api/Playback/BaseStreamingService.cs]: I don’t think this line should be in info, maybe make it more user friendly.

Comment (dkanada) [reply MediaBrowser.Api/Playback/BaseStreamingService.cs]: Why not?

Comment (Bond-009) [reply MediaBrowser.Api/Playback/BaseStreamingService.cs]: `StartFfMpeg()` the function name

Comment (Artiume) [reply MediaBrowser.Api/Playback/BaseStreamingService.cs]: I can see how it looks in the logs now as an end user and see if just saying FFmpeg starts/stops works well

**Comment by Artiume** [comment]
we can cancel the PR, but at the same time, anything with the ffmpeg engine probably warrants exposure. They're single-liners that can help show the logic process in case something goes wrong.

**Comment by JustAMan** [comment]
There are a few more lines about starting/stopping that go to info. These ones are literally for debugging some particular case.

**Comment by Artiume** [comment]
i'll close the PR. I'll open up another if I see anything else ;)

**Pull request closed without merging**